### ISSUE TITLE
ofGLUtils: "#if not defined..." is not standard C++ and produces errors in Windows

### ISSUE THREAD

**Issue by c-mendoza** [openframeworks/openFrameworks#7466]
In ofGLUtils.cpp:91, the usage of `#if not defined TARGET_OPENGLES || defined TARGET_EMSCRIPTEN` triggers a compilation error in Windows, because that usage of `defined` is non-standard. A regular desktop app will compile with a warning, but I suspect that the user targets OpenGL ES or Emscripten there might be a weird compilation or runtime problem.

The solution would be something like:
```
#if !defined(TARGET_OPENGLES) || defined(TARGET_EMSCRIPTEN)
```

But I am unsure of what the original intent of the code was. 

Is it:
```
#if !(defined(TARGET_OPENGLES) || defined(TARGET_EMSCRIPTEN))
```

Or is it:
```
#if !defined(TARGET_OPENGLES) || defined(TARGET_EMSCRIPTEN)
```
?

If the second one is the correct interpretation, I have a pull request that implements it here #7467

**Comment by NickHardeman** [comment]
if it is not OPENGLES then execute.
If it is OPENGLES and EMSCRIPTEN then execute.
So I think it would be the second one. :)

**Comment by c-mendoza** [comment]
Then #7467 should fix it!

**Comment by NickHardeman** [comment]
closed by #7467
